api/API_OVERVIEW.md

```diff
@@ -76,8 +76,7 @@ See [here](CONTRIBUTING.md#api-changes) for the v2 API change process.
 Unless otherwise stated, the APIs with the same names as v1 APIs have a similar role.
 
 * [Cluster Discovery Service (CDS)](envoy/api/v2/cds.proto).
```

Comment:
This API supports dynamically fetching cluster manager members.  See the CDS documentation

with a link to https://www.envoyproxy.io/docs/envoy/latest/configuration/cluster_manager/cds.html